pkg/capture/capture_manager.go

```diff
@@ -96,6 +102,17 @@ func (cm *CaptureManager) captureNodeHostName() string {
 	return os.Getenv(captureConstants.NodeHostNameEnvKey)
 }
 
+func (cm *CaptureManager) captureStartTimestamp() (time.Time, error) {
+	tmp := os.Getenv(captureConstants.CaptureStartTimestampEnvKey)
+
+	timestamp, err := captureUtils.ConvertStringToTimestamp(tmp)
+	if err != nil {
+		cm.l.Error("Failed to parse the timestamp into the right format", zap.Error(err))
+		return time.Time{}, err
```

Comment:
It's best to either log or return errors wrapped with context to someone that will log them for you. The latter is generally preferred in the Go community. `(github.com/pkg/errors).Wrap` is what this team has decided to use for that.

Reply:
Added wrapper